docs/robots/indoor_robots/boxer/integration_boxer.mdx

```diff
@@ -55,7 +55,7 @@ Contact [Support](#support) if you need more information.
 Ensure that the Boxer is shut off and the circuit breaker is turned to the _off_ position before performing any electrical work on the Boxer.
 
 The Boxer is equipped with an attachment interface that consists of a Power-over-Ethernet (PoE) RJ45 bulkhead connector, a USB Type-A bulkhead connector, and a 37-position panel receptacle and mated plug.
-To connect with the Boxer attachment interface see Section 10 of the [OTTO 100 V2.4 Operation and Maintenance Manual](https://help.ottomotors.com/docs/files/53683025/99909709/1/1606232510624/OMM-000094-Operation+and+Maintenance+Manual+OTTO+100+V2.4_A.pdf).
+To connect with the Boxer attachment interface see Section 10 of the [OTTO 100 V2.5 Operation and Maintenance Manual](https://docs.ottomotors.com/PDFs/100-v2.5-OMM.pdf).
```

Comment:
Do all Clearpath Customers get access to OTTO's support centre?

Reply:
All Boxer customers are required to sign the EULA and have access to OTTO's support centre. That said, I think all the links, with the exception of TalentLMS, don't require user authentication.